lerobot/common/datasets/push_dataset_to_hub/aloha_hdf5_format.py

```diff
@@ -196,6 +197,7 @@ def to_hf_dataset(data_dict, video) -> Dataset:
 def from_raw_to_lerobot_format(
     raw_dir: Path,
     videos_dir: Path,
+    codebase_version: str | None = CODEBASE_VERSION,
```

Comment:
I dont think we should give a choice.
```suggestion
```

Reply:
Fixed https://github.com/huggingface/lerobot/pull/318/commits/49c1d4627f6a6d3847e498bd494ef96925e3b7ab